doc/usecases/bindings/UC_drop_in_application.md

```diff
@@ -0,0 +1,17 @@
+# Use Case: Use libelektra as drop-in Replacement
+
+## Summary
+
+- **Scope:** API
+- **Level:** Kite
+- **Actors:** Administrator
+- **Brief:** Administrator wants to use libelektra in an application which uses another configuration management than libelektra
+- **Status:** Draft
```

Comment:
Are you sure that there is currently no implementation that fulfills this use case?